OloEditor/src/OloEditorApp.cpp

```
@@ -8,24 +8,24 @@
 
 namespace OloEngine
 {
-	class OloEngineEditor : public Application
-	{
-	public:
-		explicit OloEngineEditor(const ApplicationSpecification& spec)
-			: Application(spec)
-		{
-			PushLayer(new EditorLayer());
-		}
+    class OloEngineEditor : public Application
```

Clang (14.0.6)

[warning] class 'OloEngineEditor' defines a default destructor but does not define a copy constructor, a copy assignment operator, a move constructor or a move assignment operator

(cppcoreguidelines-special-member-functions)